01-Login/.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
remove this file

Reply:
It was in the .gitignore, somehow it wasn't ignored. I'll remove it!